string name
geometry_msgs/PoseStamped waypoint
---
bool success
